benchmark/runner/runner.sh

```diff
@@ -0,0 +1,12 @@
+cd ../exhaustive-processor
+./gradlew :processor:publishToMavenLocal
+cd ..
+mkdir tmp
+cd tmp
+git clone https://github.com/inorichi/tachiyomi.git
```

Comment:
Can you add a TODO comment that instead of installing to maven local, install to tmp/repo? so that it doesn't leave a state across runs.